tests/common_tabular_tests.py

```diff
@@ -200,7 +200,7 @@ def _verify_explain_model_local_common(self, model, x_train, x_test, y_train, y_
         assert not explanation.is_raw
         # Validate data has global info
         global_data = explanation.data(key=-1)
-        assert(InterpretData.OVERALL in global_data)
+        assert (InterpretData.OVERALL in global_data)
```

Comment:
Do we need the braces around the assert condition? Same comment for all the assert statements in this review.

Reply:
done, removed